ocs_ci/ocs/resources/deployment.py

```diff
@@ -52,7 +52,7 @@ def replicas(self):
             int: Number of replicas
         """
         self.reload()
-        return self.data.get("status").get("replicas")
+        return self.data.get("spec").get("replicas")
```

Comment:
```suggestion
        return self.data.get("status").get("replicas", 0)
```

Reply:
I actually think that the property "number of replicas" should be defined as number of replicas by spec, as it's more reliably accurate than the current status, which might be very temporary. If we need the current number as well, we can have a separate function for that, but that's out of scope for this PR.

Reply:
This would change behavior of the property value. You wouldn't get actual number of replicas. You would get only desired number of replicas but if there was any issue then you wouldn't know and you would break tests that count on this behavior.

Reply:
If searched with 'grep -ir "\.replicas " .' in ocs-ci project and it isn't used anywhere. It also makes more sense, as we also have "available_replicas" property where the actual number can be found. I'll update the description of this property to make it unambiguous.

Reply:
Ok